docs/diagrams/DeleteSequenceDiagram.puml

```diff
@@ -3,66 +3,72 @@
 
```

Comment:
Not very sure but I think the name of this puml should be DeleteStudentSequenceDiagram.puml?